web.md

```diff
@@ -0,0 +1,37 @@
+# Web
+
+**Prerequisite knowledge**: 
+
+## Intro
+Web - short for World Wide Web.
```

Comment:
Hmm I don't think Web is the short of World Wide Web. WWW is the abbreviation of World Wide Web.